### ISSUE TITLE
Importing a JSON task without a `created_at` field creates a task with a broken ULID

### ISSUE THREAD

**Issue by robwhitaker** [ad-si/TaskLite#8]
```
$ cat wop.txt
{
"body": "Buy milk",
"kanban-state": "backlog"
}
$ tl import < wop.txt
Importing … 📥 Imported task "Buy milk" with ulid "00000000000003nvz9b8xkqkr4"

$ tl
Id   Prio  Opened UTC  Body
Id "00000000000003nvz9b8xkqkr4" is an invalid ulid and could not be converted to a datetime
```

**Comment by ad-si** [comment]
Oh, wow, thanks for the heads up. Strange that I  didn't stumble upon this myself yet, but there is something weird going on:

```sh
tl ulid2utc 0000000000CDD44K7NE6JH9XG2
tl ulid2utc 0000000001CDD44K7NE6JH9XG2
tl ulid2utc 0000000002CDD44K7NE6JH9XG2
tl ulid2utc 0000000003CDD44K7NE6JH9XG2
tl ulid2utc 0000000004CDD44K7NE6JH9XG2
tl ulid2utc 0000000005CDD44K7NE6JH9XG2
tl ulid2utc 0000000006CDD44K7NE6JH9XG2
tl ulid2utc 0000000007CDD44K7NE6JH9XG2
tl ulid2utc 0000000008CDD44K7NE6JH9XG2
tl ulid2utc 0000000009CDD44K7NE6JH9XG2
tl ulid2utc 0000000010CDD44K7NE6JH9XG2  # 1970-01-01 00:00:00.032
tl ulid2utc 0000000011CDD44K7NE6JH9XG2  # 1970-01-01 00:00:00.033
tl ulid2utc 0000000012CDD44K7NE6JH9XG2  # 1970-01-01 00:00:00.034
…
```

They all should be valid ULIDs. I'll have to investigate further …

**Comment by ad-si** [comment]
Fixed this in 258df47.  I guess this was the fair punishment for using `read`.  I accidentally read a `Maybe Int64` instead of just the `Int64`. 

Me:

>  I'll use Haskell and it's awesome type system to write better and safer code

Also me: 

> I'll use `read` to circumvent all safety features of the type system

I'm just banging my head against the wall right now over how stupid I am 🤦‍♂️.

And don't get me started on how silly that code was in the first place, even if I had implemented it correctly 😩.

On making it better the next time 🥂😅.
